### PR TITLE
Nginx shim for local_vars_big

### PR DESCRIPTION
**Pull request by georgejhunt**
Tested on RPI 4, and vm's(buster,ubunti18).

This started out as a shim, proxying everything to apache.
But now there are many services served directly by nginx (see https://github.com/georgejhunt/iiab/blob/nginx/roles/nginx/files/README.md)

### CODE REVIEW

File: roles/nginx/templates/kalite-nginx.conf [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# Upstream KA-Lite server uses port 7007
+# Nginx proxy for KA-Lite uses port 8008
+#
+# If you want the website to be accessible at a different port, change
+#  PROXY_PORT = nnnn setting in /var/ka-lite/.kalite/settings.py
+# and change the below accordingly.
+
+
+upstream kalite {
+    server 127.0.0.1:7007;
```
Comment (jvonau) [inline roles/nginx/templates/kalite-nginx.conf:12]: what sets the port used at 7007 when kalite_server_port: 8008 is in default_vars.yml?

File: vars/default_vars.yml [hunk 2]
```diff
@@ -216,6 +220,24 @@ exFAT_enabled: True
...
+apache_interface: "127.0.0.1"
+apache_install: True
+apache_enabled: True
+# The following variable, if True, allows Admin Console to poweroff IIAB
+allow_apache_sudo: False
+
+nginx_port: "80"
+nginx_interface: "0.0.0.0"
+nginx_install: True
+nginx_enabled: True
```
Comment (jvonau) [inline vars/default_vars.yml:239]: below kalite_server_port: should be set to 7007

**Review by jvonau — COMMENTED** [review]
see comments inline below

File: roles/kolibri/tasks/main.yml [hunk 3]
```diff
@@ -159,7 +171,7 @@
       value: "{{ kolibri_url }}"
     - option: kolibri_exec_path
       value: "{{ kolibri_exec_path }}"
-    - option: kolibri_http_port
+    - option: kolibri_port
```
[the remark this answers is not in the document]
Comment (jvonau) [reply roles/kolibri/tasks/main.yml:174]: Based on the migrated list in the README, is there a missing kolibri-nginx.conf for the proxy_pass to apache or to proxy the kolibri_url to the port directly?

File: roles/nginx/templates/uwsgi.unit [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Unit]
```
Comment (jvonau) [inline roles/nginx/templates/uwsgi.unit:1]: missing [

**Comment by georgejhunt** [comment]
Thanks Jerry

**Comment by georgejhunt** [comment]
incorporated in gh-nginx-e

**Pull request closed without merging**